tests/roles/ceph_migrate/tasks/mon.yaml

```diff
@@ -169,53 +178,23 @@
 - name: Redeploy MON
   become: true
   block:
-    # when a label is added, if we're not fast enough, a mon is automatically
-    # added on the storage network. However, in case the node has multiple ip
-    # addresses, it might happen that the mon is deployed using the right IP.
-    # For this reason we need to redeploy it by rm + add (as redeploy does not
-    # accept an IP as input
-    - name: MON - Check quorum
-      ansible.builtin.command: "{{ ceph_cli }} mon stat -f json"
-      register: monstat
-      retries: 20
-      delay: 3
-      until: "'{{ target_node.split('.')[0] }}' in monstat.stdout | from_json | community.general.json_query('quorum[*].name') | default([]) | list"
-      loop_control:
-        label: "MON - Check quorum"
-
-    # Even though we explicitly redeploy a given mon using the host:ip format,
-    # it is possible that the orchestrator (who owns the process and the spec)
-    # takes initiative to redeploy a mon, on an arbitrary ip address (if many)
-    # between the `ceph orch rm ...` command and the `ceph orch daemon add ...`
-    # one. For this reason, the deamon redeploy follows this flow:
-    # 1. update and unmanage the spec
-    # 2. redeploy on the right network as per the official doc:
-    #    https://docs.ceph.com/en/quincy/cephadm/services/mon/
-    # 3. manage the spec
-    # Note: a pause between the steps is required to allow the orchestrator to
-    #       process the spec and update the daemons ref
-    - name: Unmanage mons
```

Comment:
Moving this task at the beginning to have a more clean flow (and easy to read / understand)